README.md

```diff
@@ -279,15 +279,15 @@ Python packages can be installed using `pip` from the [Python Package Index](htt
 ### Accessing C++ function from Python
 
 To make a web application in Python, the C++ functions need to be called somehow.
-Python can call functions in a C++ library if it's functions use [Python.h datatypes](https://docs.python.org/3.7/extending/index.html). This requires a lot of boilerplate and conversions, several tools are out there that make the boilerplate/conversions much simpler. The tool we chose to use is [pybind11](https://github.com/pybind/pybind11) as it is currently (May 2019) activly maintained and is header only library.
+Python can call functions in a C++ library if its functions use [Python.h datatypes](https://docs.python.org/3.7/extending/index.html). This requires a lot of boilerplate and conversions, several tools are out there that make the boilerplate/conversions much simpler. The tool we chose to use is [pybind11](https://github.com/pybind/pybind11) as it is currently (May 2019) actively maintained and is a header-only library.
 
 To use pybind11, it must installed with pip
 
 ```{.awk #pip-pybind11}
 pip install pybind11
 ```
 
-Pybind11 requires a bindings (PYBIND11_MODULE macro) to expose C++ constants/functions/enumerations/classes to Python. The bindings can be compiled to a shared library (eg. pybubble.so) which can be imported into Python.
+Pybind11 requires a bindings macro (PYBIND11_MODULE) to expose C++ constants/functions/enumerations/classes to Python. The bindings can be compiled to a shared library (eg. pybubble.so) which can be imported into Python.
```

Comment:
not sure this is what you meant but ok

Reply:
Put macro in own sentence in https://github.com/NLESC-JCER/cpp2wasm/pull/17/commits/f68628c6165ab16bbc557bd5b3f62f166e87f2f4